quarkus-jnosql-mongodb/src/main/java/org/jnosql/demoee/DevelopersResource.java

```diff
@@ -22,7 +22,7 @@ public List<Developer> listAll() {
         return template.select(Developer.class).result();
     }
 
-    @Path("/byName")
+    @Path("/findByName")
```

Comment:
In the rest API nomenclature, does it make sense?

Reply:
I'm fixing it. Thanks, @otaviojava!